### PR TITLE
Feature/rewrite result UI

### PR DESCRIPTION
**Pull request by nddmanh**
UI result:
image

### CODE REVIEW

File: src/modules/playRoom/components/ResultRoll.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      id: 2,
+      src: "./image-pick/ca.png",
+      value: "Ca",
+    },
+    { 
+      id: 3,
+      src: "./image-pick/cua.png",
+      value: "Cua",
+    },
+    // {
```
Comment (ppppbn) [inline src/modules/playRoom/components/ResultRoll.js:20]: Remove comments

File: src/modules/playRoom/components/ResultRoll.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+
+const ResultRoll = () => {
+  const [imagesResultRoll, setImagesResultRoll] = useState([
+    {
```
Comment (ppppbn) [inline src/modules/playRoom/components/ResultRoll.js:5]: Add a small comment here to notice everyone that this is fake data

File: src/modules/playRoom/components/ResultRoll.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  ]);
+  return (
+    <div className="previous-results">
+      <div className="head-result">
+        <h3>KẾT QUẢ</h3>
+      </div>
+      <div className="result">
+        <div className="result-box">
+          {imagesResultRoll.map((result) => (
+            <input type="image" className='img-result' src={result.src} alt="" name="" value={result.value} />
```
Comment (ppppbn) [inline src/modules/playRoom/components/ResultRoll.js:45]: The result should not be an input isn't it?

File: src/modules/playRoom/pages/PlayRoom.css [hunk 4]
```diff
@@ -190,3 +190,47 @@
...
+    align-items: center;
+}
+
+.img-result {
+    margin: 40px 10px 0px 10px;
+    width: 75px;
+    cursor: pointer;
+    border:  1px solid rgba(0,0,0,0.25);
+    border-radius: 10px;
+    transition: 0.35s;
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.css:235]: Why do you leave only 0.35s for transition property?

File: src/modules/playRoom/components/ChatBox.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/ChatBox.js:1]: Remove React

File: src/modules/playRoom/components/CountDown.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/CountDown.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:1]: Remove React

File: src/modules/playRoom/components/PickItem.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:22]:
```suggestion
      <div className="select" onClick={showModal} >                
```

File: src/modules/playRoom/components/PickItem.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+  const handleCancel = () => {
+    setIsModalVisible(false);
+  };
+  
+
+  return (
+    <>
+      <div class="select"  onClick={showModal} >                
+        <input type="image" className="img-select" src={img.src} alt="" name="" value={img.value} />
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickItem.js:23]:
```suggestion
        <input type="image" className="img-select" src={img.src} alt="pick item" value={img.value} />
```

File: src/modules/playRoom/components/PickList.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:1]: Remove React

File: src/modules/playRoom/components/PickList.js [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+import { Row, Col } from 'antd';
+import PickItem from './PickItem';
+
+const PickList = () => {
+  const [imagesPick, setImagesPick] = useState([
```
Comment (lethemanh) [inline src/modules/playRoom/components/PickList.js:6]: Move ```imagesPick``` to constant file

File: src/modules/playRoom/components/ResultRoll.js [hunk 12]
```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
```
Comment (lethemanh) [inline src/modules/playRoom/components/ResultRoll.js:1]: Remove React

File: src/modules/playRoom/pages/PlayRoom.js [hunk 13]
```diff
@@ -1,8 +1,25 @@
 import React, { useState, useEffect } from 'react';
```
Comment (lethemanh) [inline src/modules/playRoom/pages/PlayRoom.js:1]: Remove ```React, { useState, useEffect }```

**Pull request closed without merging**